README.md

```diff
@@ -346,13 +348,13 @@ Each concept exercise and concept have an `introduction.md` file. If you want th
 
 Concept placeholders must use the following format:
 
-```
+```text
 %{concept:<slug>}
 ```
 
 For example, if the track has a concept named `floating-point-numbers` then an `introduction.md.tpl` file can contain:
 
-```
+```text
```

Comment:
@wolf99 I notice that #462 used `markdown` instead as the language for these. Is that better?

Reply:
This is based on the converstation on Slack here: https://exercism-team.slack.com/archives/CAQP7JL3T/p1635445165124700

I dont think the discussion was conclusive though - `text` could be good either.

Reply:
Thanks.

Let's go with `text` here. `configlet generate` uses custom templating syntax - even if it might have the same syntax as some other templating language, it's confusing to suggest that configlet actually uses that language internally. And we'd barely benefit from syntax highlighting in this code block anyway.